integration/runner/runner.go

```diff
@@ -68,6 +69,14 @@ func getAttributes(ipAddress, portStr string) (*cadvisorApi.Attributes, error) {
 	return &attributes, nil
 }
 
+func getIp(host string) (string, error) {
+	if *sshConfig == "" {
+		return host, nil
+	}
+	out, err := exec.Command("awk", fmt.Sprintf("/Host %s/ {getline; print $2}", host), *sshConfig).CombinedOutput()
```

Comment:
one would hope, but I wasn't able to find a better solution on SO or the man pages.  Happy to change if anyone with more ssh knowledge could suggest a better way.